fast-sim-and-analysis/fccanalyses/doc/starterkit/FccFastSimAnalysis/Readme.md

```diff
@@ -232,30 +239,31 @@ legend['VV'] = 'VV boson'
 ```
 
 And run the plots (the line below is temporary, will be updated once we properly link the env var) 
```

Comment:
could you also remove the comment in parenthesis?

Reply:
good catch, thanks